### ISSUE TITLE
Allow highlighting of multiple lines and not just series of lines

### ISSUE THREAD

**Issue by incertia** [isaacs/github#453]
If I wanted to reference specifically lines 5, 11, 24-33, and 68, I currently can only do this with #L5-L68, which highlights all lines in that range. We should be able to add comma separated ranges (e.g. #L5,L11,L24-L33,L68).

**Comment by jamen** [comment]
+1

**Comment by munick** [comment]
+1

**Comment by javjar** [comment]
+1

**Comment by bjadamson** [comment]
+1

**Comment by gmkohler** [comment]
+1

**Comment by vedz91** [comment]
👍

**Comment by andrii-kovalenko-ct** [comment]
+1

**Comment by bjadamson** [comment]
This was back from my days in corporate america, so maybe it'll resonate?

ping!!

**Comment by bam0382** [comment]
+1

**Comment by sholladay** [comment]
I needed this today to highlight an unnecessary function. It would have been very useful to be able to highlight its definition and its callsite. There is other code in between and selecting the whole range looks very confusing for the person I am trying to share the highlight with.

**Comment by designerdean** [comment]
+1

**Comment by JaoNoctus** [comment]
+1

**Comment by itmustbejj** [comment]
👍

**Comment by jamen** [comment]
y'all would probably enjoy [this issue](https://github.com/isaacs/github/issues/9) too :wink:

**Comment by karlsnyder0** [comment]
+1

**Comment by jenPlusPlus** [comment]
+1

**Comment by sweeneyrobb** [comment]
+1

**Comment by FocuzJS** [comment]
+2

**Comment by yogeshmuthuraj** [comment]
Will there be a feature for this?

**Comment by ravenpi** [comment]
Agreed.  I'm trying to write some documentation, and it would be terrific if I could highlight multiple individual lines (e.g., version numbers and build numbers) in a single file, and not have a bunch of separate links to the file.

**Comment by chrisblossom** [comment]
Any updates?

**Comment by ycjcl868** [comment]
+1

**Comment by NobleDraconian** [comment]
Heya, it's been about 2 years - any updates on this? I have a need for this as well.

**Comment by derek-smith** [comment]
Bitbucket has this and I use it all the time at work. You can highlight every place a specific variable is used, for instance. Super useful.

**Comment by akshaynagpal** [comment]
Any update on this?

**Comment by drkohlipk** [comment]
It's been over 5 years and this is a vital feature that nearly every other repo management service has, can we please implement this?

**Comment by Levi-Lesches** [comment]
Reminder that this is _not_ the official GitHub feedback forum, this is a community-run collection of issues. 

To involve GitHub staff, open a discussion in the [official GitHub repository](https://github.com/github/feedback/discussions/categories/general-feedback).

**Comment by fathur** [comment]
@Levi-Lesches Is there anyone post this issue on official Github repository? If exists, could you share it?

**Comment by TPS** [comment]
I didn't see any myself @ the feedback repo, so you're welcome to open 1 up (but do link back here). 🙇🏾‍♂️

**Comment by xro** [comment]
GitHub c'mon!! How one can indicate parts of the code to the team mates when cannot select non consecutive lines? Maybe by sending multiple links with block of code selected????
I thought ability of browsing code is one of the most important with GitHub, isn't it? 
Attlasian Stash has it since ages.

**Comment by incertia** [comment]
It is probably worth opening an issue on the official Github tracker. I'm too lazy at the moment but if anyone wants to do it feel free.

**Comment by TPS** [comment]
@incertia I think ~~someone~~ @xro 🙇🏾‍♂️ ([from 2 comments before](https://github.com/isaacs/github/issues/453#issuecomment-896576524)) _already_ has.…

@All Please upvote & comment @ https://github.com/github/feedback/discussions/5022

**Comment by incertia** [comment]
> @incertia I think ~~someone~~ @xro 🙇🏾‍♂️ ([from 2 comments before](https://github.com/isaacs/github/issues/453#issuecomment-896576524)) _already_ has.…

Well there it is...
